pacman/makepkg-mingw

```diff
@@ -86,9 +86,18 @@ for _mingw in ${MINGW_INSTALLS}; do
       _arch=x86_64
       _msystem=MINGW64
     ;;
+    arm64)
+      _arch=arm64
+      _msystem=ARM64
+    ;;
   esac
 
-  if [ -f "/${_mingw}/bin/gcc.exe" ]; then
+  # Currently, arm64 is only supported through cross-compilation from mingw32
+  if [ "${_mingw}" = "arm64" -a -f "/mingw32/bin/gcc.exe" ]; then
+    MSYSTEM=${_msystem} \
+    PATH=/mingw32/bin:$(echo $PATH | tr ':' '\n' | awk '$0 != "/opt/bin"' | paste -sd:) \
```

Comment:
Something like `PATH=/arm64/bin:mingw32/bin:$(echo ....` might also work, but I'm currently not sure what the implications of this would be. I assume it would first look for a native binary in `/arm64/bin`, and if that fails, look at `/mingw32/bin` instead.